tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    /**
+     * Must allow:
+     */
+    "downlevelIteration": true,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "noUnusedParameters": false,
+    "removeComments": false,
+    "incremental": true,
+    /**
+     * Nice to have:
+     */
+    // "preserveConstEnums": true,
```

Comment:
I'm fine if you uncomment it.😊

Reply:
We can't uncomment without massive changes across the repo. Lets address this in a followup